index.html

```diff
@@ -0,0 +1,177 @@
+            <p>
+                The new FIFA is modernising football to be global, accessible and inclusive in all aspects. Not just on one or two continents, but everywhere.
+            </p>
+        </div>
+        <div class="headlin-description">
+            <h2 class="second">2022.12.18(TUE) ~ 20(WED)</h2>
+            <p>@ Redmond, Doha, Qatar, WORLD CUP</p>
+        </div>
+    </section>
+    <section class="main-program" id="Program">
+        <div class="title-program">
+            <h3>Main program</h3>
+        </div>
+        <div class="cards">
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/champions.png" alt="champions">
+                </div>
+                <div class="title-card">
+                    <h4>champions</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/laliga.png" alt="laliga">
+                </div>
+                <div class="title-card">
+                    <h4>La liga</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/EnglishPremierLeague.webp" alt="premier">
+                </div>
+                <div class="title-card">
+                    <h4>premier league</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/serie.jpg" alt="serie">
+                </div>
+                <div class="title-card">
+                    <h4>serie A</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="featured" id="Featured">
+    
+    </section>
+
+    <section class="partner" id="Studio">
+        <h3>Partner</h3>
```

Comment:
[REQUIRED] While giving this text. a margin you must have added it to the sides too instead of just the top.

![image](https://user-images.githubusercontent.com/62938735/205462682-0d9efed6-2ec8-48ee-8417-e0100c61fef9.png)